ecobee/beam_batch.py

```diff
@@ -0,0 +1,144 @@
+import argparse
+import logging
+import os
+from collections import namedtuple
+from datetime import datetime
+from time import time
+
+import apache_beam as beam
+from apache_beam.io import ReadFromText
+from apache_beam.io import WriteToText
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+
+DataPoint = namedtuple("DataPoint", ["time", "action"])
+Sequence = namedtuple("Sequence", ["seq_len", "datapoints"])
+
+timestamp_prefix = "TS:"
+timestamp_postfix = "GMT "
+action_prefix = "Action:"
+date_fmt = "%Y-%m-%d %H:%M:%S %z"
```

Comment:
Capitalize these they are constants. It's just a python convention.

Reply:
done